### ISSUE TITLE
SonarLint on PDSOE 11.7 - Fails with JRE version

### ISSUE THREAD

**Issue by gquerret** [Riverside-Software/sonar-openedge#1031]
On behalf of AMX

When using SonarLint with PDSOE 11.7 in connected mode (SonarQube 9.4), following message is displayed in SonarLint console:
```
Unable to start the SonarLint engine
java.lang.IllegalStateException: The plugin [kotlin] does not support Java 1.8.0_101
at org.sonarsource.sonarlint.core.plugin.PluginInstancesLoader.instantiatePluginClasses(PluginInstancesLoader.java:145)
at org.sonarsource.sonarlint.core.plugin.PluginInstancesLoader.load(PluginInstancesLoader.java:75)
at org.sonarsource.sonarlint.core.plugin.PluginRepository.start(PluginRepository.java:57)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
...
```

This can also happen with different plugins (xml, tsql, ...).

**Comment by gquerret** [comment]
PDSOE 11.7 requires Java 1.8, and can't use Java 11 (as it relies on a very old Eclipse version). As SonarLint is executed in the same process as Eclipse, it is also executed with Java 1.8.

In connected mode (which is the standard for OpenEdge and CABL), SonarLint fetches the plugins available on the server side, and tries to execute them locally. Starting from SonarQube version 9, some plugins are not compatible with Java 1.8 anymore, and require Java 11. Kotlin is one of them.

In order not to fail in PDSOE 11.7, a new beta version of SonarLint is available here: https://dl.rssw.eu/org.sonarlint.eclipse.site-6.2.2220.zip

This version excludes some plugins from the analysis, so that it's possible to use SonarLint on PDSOE 11.7 while connected to SonarQube 9.9 LTS. It is confirmed that some commercial 9.x (before 9.9 LTS) editions have a problem with the license plugin which is also compiled with Java 11. This has been fixed by SonarSource. If you're running 9.x, you should upgrade to SonarQube 9.9 LTS.

Note that this SonarLint version also includes the fix for #1025.
Feedback appreciated !

**Comment by gquerret** [comment]
Changes will be included in next release.
